plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -263,7 +263,7 @@
             description: Mode of power assignment for the 2.4GHz radio interface. Accepts "Global" or "Custom". For example, "Custom".
             type: str
             required: False
-          power_level:
```

Comment:
Do we need to change here or in the code?

Reply:
In the code i have used powerlevel, hence the api response of AP config also same. i have fixed the code long back forget to change in this document.

Reply:
If power_level is released to customer, then we can use power_level itself. 
power_level is more consistent with YAML's preference for clarity and common conventions.

Why did you change power_level into powerlevel in the code?

Reply:
When we merged first time we used the power_level QA bug was raised not working in power level. then we checked the code. while checking we found update ap config the payload was changed to powerLevel.  actually in the payload as well as ap config response, both powerlevel only using. we are using function to convert snake case to camel case to create payload. there we are getting problem. those we fixed in the last bug with powerlevel. that i need to update in the yml document i missed it